specification/communication/data-plane/CallAutomation/preview/2024-01-22-preview/communicationservicescallautomation.json

```diff
@@ -1882,6 +2167,10 @@
         "transcriptionConfiguration": {
           "$ref": "#/definitions/TranscriptionConfiguration",
           "description": "Live Transcription Configuration."
+        },
+        "mediaStreamingOptions": {
```

Comment:
this should be mediaStreamingConfiguration based on this PR https://skype.visualstudio.com/SCC/_git/sync_calling_concore-platform/pullrequest/1093927